htmd/data/test-param/benzamidine_esp_freeze_restart/parameters/CGenFF_2b6/B3LYP-6-31Gstar-vacuum/mol.psf

```diff
@@ -17,10 +17,10 @@ PSF
       12 L    1    MOL  H5   HG61       0.0657262       0.0           0
       13 L    1    MOL  N1   N2P1             0.0       0.0           0
       14 L    1    MOL  N2   N2P1             0.0       0.0           0
-      15 L    1    MOL  H6   HGP2        0.240023       0.0           0
-      16 L    1    MOL  H7   HGP2        0.240023       0.0           0
-      17 L    1    MOL  H8   HGP2        0.240023       0.0           0
-      18 L    1    MOL  H9   HGP2        0.240023       0.0           0
+      15 L    1    MOL  H6   HGP2        0.327235       0.0           0
```

Comment:
Why do only these charges change? Doesn't this mean the total charge of the molecule change? Seems weird.

Reply:
This test is still disable in this PR, so it doesn't matter.

Reply:
OK, but we'll need to revise afterwards